sema/ro/creator/api.py

```
@@ -0,0 +1,89 @@
+        pass
+
+    def __call__(self, uses: dict[str, any]) -> RocStrategy:
+        """Extra call allows for configuration of the strategy."""
+        return self
+
+
+def read_roccfg(rocyml: Path) -> RocModel:
+    context: dict = os.environ.copy()
+    loader = LoaderBuilder().to_resolve(context).build()
+    with rocyml.open() as f:
+        roccfg = yaml.load(f, Loader=loader)
```

Ruff (0.8.2)

Probable use of unsafe loader `loader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)